src/classes/BGE_BatchGiftEntry_UTIL.cls

```diff
@@ -163,10 +160,48 @@ public with sharing class BGE_BatchGiftEntry_UTIL {
         return fieldApiNames;
     }
 
+    private static GE_Template.Template deserializeTemplate(DataImportBatch__c batch) {
+        GE_Template.Template template = (GE_Template.Template) JSON.deserialize(
+                batch.Form_Template__r.Template_JSON__c,
+                GE_Template.Template.class
+        );
+        return template;
+    }
+
+    private static String getNewBatchGiftEntryVersionFields() {
+        return DataImport__c.Additional_Object_JSON__c.getDescribe().getName();
+    }
+
+    private static Boolean isNewBatchGiftEntryVersion(DataImportBatch__c batch) {
+        return batch.Batch_Gift_Entry_Version__c >= 2.0;
```

Comment:
This is something realllyyyyy minor...

Can you imagine a future scenario where would ever need to increment this number value? If so - perhaps we could consider moving it to a constant at the header of the class, or somewhere else.

Reply:
@sokosaurus This version number shouldn't ever change. Since it's only used in one place in the class we probably don't need a constant at this point I don't think.